packages/react-component-library/src/components/Popover/Popover.tsx

```diff
@@ -91,6 +95,8 @@ export const Popover: React.FC<PopoverProps> = ({
             onMouseLeave={handleOnMouseLeave}
             position={PLACEMENTS.ARROW_POSITION}
             scheme={scheme}
+            aria-describedby={contentId}
+            contentId={contentId}
```

Comment:
My comment seems to have been lost but it was something along the lines of:

> Is this right that the `aria-describedby` is referencing itself? I would have thought it would clone `content` with an ID attribute which is then referenced or if `content` is always a `string` then it is not referenced at all.

Reply:
It passes contentId into the `FloatingBox` component where it is then applied to a child element within the sub-component so it's actually referencing a direct descendant.

FloatingBox:

```
FloatingBox
----div described-by="contentId"
--------div id="contentId"
```

Reply:
I misinterpreted this change. Cool 👍